### PR TITLE
Location secure redirect

### PR DESCRIPTION
**Pull request by variousred**
multidomain clients have location level secure_domain settings,
prior to this PR they were only respecting the client level global setting

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -11,7 +11,7 @@ GIT
 
 GEM
   remote: https://rubygems.org/
-  remote: https://msc9777J2TEEbgRsedKE@gem.fury.io/g5dev/
+  remote: https://gem.fury.io/g5dev/
```
Comment (thescubageek) [inline Gemfile.lock]: Are we certain this won't break anything?

**Comment by thescubageek** [comment]
Looks good, see comment ^^. Remeber to bump subversion to 1.1.5 in version.yml and let's get this fix out tomorrow.

**Comment by levibrown** [comment]
:shipit:

**Comment by riverblufftrail** [comment]
@variousred great work. Thanks for your quick response to this issue!

**Comment by jcstringer** [comment]
Looks good nice work

**Comment by variousred** [comment]
tested on my single domain client mjsingledomain, and on compass rock

**Comment by variousred** [comment]
@ceckert were you having problems with the gemfile.lock set up like this?

**Pull request merged**